analytics/lib/counts.py

```diff
@@ -58,11 +75,15 @@ def process_count_stat(stat, fill_to_time):
 
     currently_filled = currently_filled + timedelta(hours = 1)
     while currently_filled <= fill_to_time:
+        logger.info("START %s  %s  %s" %(stat.property, stat.interval, currently_filled))
         FillState.objects.filter(property = stat.property) \
                      .update(end_time = currently_filled, state = FillState.STARTED)
+        start = time.time()
         do_fill_count_stat_at_hour(stat, currently_filled)
+        end = time.time()
```

Comment:
These start and end should include the FillState queries as well; i.e. they can go from right after the first logging command to right before the second logging command.